plugins/backstage-plugin-coder/src/plugin.ts

```diff
@@ -58,16 +58,6 @@ export const CoderProvider = coderPlugin.provide(
   }),
 );
 
-export const CoderAuthWrapper = coderPlugin.provide(
-  createComponentExtension({
-    name: 'CoderAuthWrapper',
-    component: {
-      lazy: () =>
-        import('./components/CoderAuthWrapper').then(m => m.CoderAuthWrapper),
-    },
-  }),
-);
-
```

Comment:
Still undecided on whether I'll be exposing the new auth components to the end user; going to wait until the vendored SDK is plugged in, so I can get a sense for whether they're really needed